runtime/codert_vm/cnathelp.cpp

```diff
@@ -3240,17 +3243,20 @@ old_slow_jitNewInstanceImplAccessCheck(J9VMThread *currentThread)
 			 * unless the two classes are in the same nest (JDK11 and beyond).
 			 */
 #if JAVA_SPEC_VERSION >= 11
-			if (NULL == thisClass->nestHost) {
-				if (J9_VISIBILITY_ALLOWED != vmFuncs->loadAndVerifyNestHost(currentThread, thisClass, 0)) {
+			J9Class *nestHost1 = thisClass->nestHost;
```

Comment:
perhaps `thisClassNestHost` and `callerClassNestHost`

Reply:
Fixed.